recipes/micronota/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+pip install burrito click
```

Comment:
yes, it is a pity that can't set pip dep. And oh yes, micronota is dependent on both - I happen to be a main developer. It is in early development stage. Just wanna to put it in for a demo tomorrow. 

if no other problem, merge it? we can set click and burrito recipes easily later, so that the "pip install" can be removed.